### ISSUE TITLE
Can join room, but no sound

### ISSUE THREAD

**Issue by bgod** [pdaddyo/soundbounce#90]
New installation on Windows 8.1 x64
Sound works in other apps (spotify desktop player, firefox, etc).
I can join a room and see the track progress bar moving.  Pause/unpause has no effect.  
Rebooted PC.  
Re-installed soundbounce.

**Comment by drawcard** [comment]
Having this same problem in Mac OSX 10.10.1, is there a log file I can post here?

**Comment by pdaddyo** [comment]
@drawcard, we think this may be related to your playlist count against your user - how many playlists do you have in spotify?  Unfortunately there's no log file on the OSX version for now, this is coming soon.  Please try again and let us know if it ends up working for you!

**Comment by drawcard** [comment]
I don't know the exact count but I know I have around 20 playlists or so at least...

**Comment by pdaddyo** [comment]
Have you guys tried again since?  This is a known issue on first launch
with libspotify, especially on OSX.  Let us know how you get on.

**Comment by bgod** [comment]
Yeah, I have too many Spotify playlists to count.  The problem comes and goes for me.  I usually just restart a few times to get around it.   The next time it happens, I'll send the log.

**Comment by drawcard** [comment]
Just gave it another go after computer reboot, still no sound output. Everything else functions though, like the progress bar and the chat room etc.

**Comment by joshuaswilcox** [comment]
Same as @drawcard on Yosemite, no sounds, restarted several times waiting in between, but everything else appears to be working

**Comment by joshuaswilcox** [comment]
So I left mine open after the third time of restarting and it eventually, after several minutes (not sure how many) started.  Not sure the root cause but patience seems to work :)

**Comment by pdaddyo** [comment]
The "no audio for first few tries" does seem to affect OSX more than Windows.  @joshuaswilcox - how many spotify playlists do you have?  We think it might be these syncing with libspotify on first few launches.

**Comment by bgod** [comment]
It seems to happen less often for me if I sit at the list of rooms for a minute or so before joining one.  Could be a coincidence, but I've been trying that since you mentioned it could be a syncing issue related to number of playlists.

**Comment by joshuaswilcox** [comment]
@pdaddyo I have about 55 playlists

**Comment by pdaddyo** [comment]
Is audio working more consistently now?

**Comment by joshuaswilcox** [comment]
Yeah once it started wiring in hasn't stopped

**Comment by EdPoole1** [comment]
I'm having the same problem on Windows - which is odd, as it has worked flawlessly for me from 1/1 through Friday (today is first time I've tried since Friday).

how would I go about finding my logfile (assuming you'd want me to post it)?

**Comment by pdaddyo** [comment]
On Windows, soundbounce-log.txt in the install folder.  I think there were some issues at Spotify's end in last few hours, my audio stopped for a bit but is now back...  Cheers.

**Comment by EdPoole1** [comment]
thanks - I did the launch as administrator thing & that worked

**Comment by bgod** [comment]
To update my previous comment, waiting a minute before joining a room after signing in seems to be fairly reliable for me.  If I join a room too quickly after signing in, I often get no sound.
